compiler/tflchef/tflite/src/Op/Less.cpp

```diff
@@ -17,20 +17,35 @@
 #include "Less.h"
 
 #include "Convert.h"
+#include "FillerHelper.h"
 
 namespace tflchef
 {
 
 void TFliteOpLess::filler(const tflite::Operator *op, TFliteImport *import,
                           tflchef::ModelRecipe *model_recipe) const
 {
-  // Nothing to do with filler
+  const auto &inputs = *op->inputs();
+
+  for (int input : inputs)
+  {
+    fill_tensor_to_import(input, import);
+  }
```

Comment:
Added due to Less can have Const input as in example for https://github.com/Samsung/ONE/pull/12319#issuecomment-1892841266